mne/datasets/utils.py

```diff
@@ -585,8 +585,6 @@ def _download_all_example_data(verbose=True):
     megsim.load_data(condition='visual', data_format='evoked',
                      data_type='simulation', update_path=True)
     eegbci.load_data(1, [6, 10, 14], update_path=True)
-    sleep_physionet.age.fetch_data(subjects=[0, 1], recording=[1],
-                                   update_path=True)
```

Comment:
I think something went wrong in your rebase

Reply:
yes @DiGyt please revert